valkey.conf

```diff
@@ -678,6 +678,9 @@ repl-diskless-sync-max-replicas 0
 # "on-empty-db" - Use diskless load only when current dataset is empty. This is 
 #                 safer and avoid having old and new dataset loaded side by side
 #                 during replication.
+# "flushdb"     - Flush the current db contents before parsing. Note that if
```

Comment:
Note for @madolson, we should clean up this documentation to make it clear this is for all databases, and not just the current DB.

Reply:
> once upon a time it was an initial way to make diskless replication - without any checks.
but it was thrown away because it's dangerous

@kronwerk Did this option exist in the past? If yes, what was the name of the option?

Reply:
Diskless load seems to have been added in 2de544cfcc6d1aa7cf6d0c75a6116f7fc27b6fd6, with only on-empty-db and swapdb as options. An earlier version of the PR https://github.com/redis/redis/pull/4491 had "flushdb" as an option, but this was never merged, thus never available to users.

It was discussed in the PR that a replica which has flushed all its data and not yet synched should not be able to be promoted to primary in a cluster. I don't know how that works but it was mentioned in comments. Do we have any such mechanism in place and/or would it be relevant now?

Reply:
> @kronwerk Did this option exist in the past? If yes, what was the name of the option?

@zuiderkwast you're right, it was proposed, but never merged, description was taken as is from that attempt

Reply:
Ok... Can you find out what is meant in this comment? https://github.com/redis/redis/pull/4491#issuecomment-349931034

I don't completely understand the protection they're talking about here. But if it's good, maybe we want it?

Reply:
> I don't completely understand the protection they're talking about here. But if it's good, maybe we want it?
I think they're talking about isUnsyncedSlave from here: https://github.com/redis/redis/commit/24240a7593e3ce904a7c46a2b8fd2770dc3e4024
but I can't find what's inside this function and thus I'm not sure whether it's useful or not, especially after such a long time